translator/cmdutil/translatorutil.go

```diff
@@ -44,6 +44,9 @@ func TranslateJsonMapToTomlFile(jsonConfigValue map[string]interface{}, tomlConf
 			fmt.Println(exitSuccessMessage)
 		}
 	} else {
+		for _, errMsg := range translator.ErrorMessages {
```

Comment:
I don't think you need to error out the message since [the executable for config-translator](https://github.com/aws/amazon-cloudwatch-agent/blob/master/cmd/config-translator/translator.go#L81-L83) has covered it

Reply:
ohh..good catch. I read through that but forgot about it before modifying. Will update this in next commit